index.html

```diff
@@ -141,620 +141,809 @@
+perform &quot;Know Your Customer&quot; checks on accounts, this credential
+can also be used as sufficient verification by other financial institutions.
+This can help Jane assure destination banks that she is verified, thereby
+allaying concerns about misdirected transactions and money laundering.
+          </dd>
+          <dt>
+            <udef>F.2 Money transfer</udef>
+          </dt>
+          <dd>
+Susan wants to send funds to her family in another country via a popular
+money transfer service. She has <a>verifiable claims</a> in her <a>credential
+repository</a> that can be used to share her <a>identity profile</a>. She has
+also been sent a <a>claim</a> from her family verifying their banking
+information. By sharing these with the money transfer service, they can
+automatically verify the source and destination of funds, thus being
+confident in the delivery of those funds and satisfying various regulations
+regarding prevention of money laundering.
+          </dd>
+          <dt>
+            <udef>F.3 Closing account</udef>
+          </dt>
+          <dd>
+John opens a checking account at Big Bank Co and is issued a verifiable claim
+indicating that the account exists, that the bank verified John's identity,
+and that John has access to the account. Some time later, John is moving to a
+new city and decides to close that account. Big Bank Co needs to revoke that
+claim as part of their normal account closing process.
+          </dd>
+          <dt>
+            <udef>F.4 Trying out a new service</udef>
+          </dt>
+          <dd>
+Nikita has several accounts with BigBank, as well as a brokerage account with
+WallStreetCo. She had placed all of her <a>claims</a> in a <a>credential
+repository</a> at BigBank that came free when she opened her accounts.
+WallStreetCo is now offering a new <a>repository</a> that has an interface
+she thinks she will prefer. Nikita copies her <a>claims</a> from BigBank into
+the repository at WallStreetCo to experiment with their service, but
+continues to use the service from BigBank while she is testing.
+          </dd>
+          <dt>
+            <udef>F.5 New bank account from home</udef>
+          </dt>
+          <dd>
+Alice wants to open a new bank account. BigOnlineBank offers the ability to
+do this from home if she can provide electronic credentials. She offers
+government-issued certificates that verify her identity (address, national
+identity number, etc.), and opens her new account from her couch.
+          </dd>
         </dl>
-    </section>
-
-    <section>
+      </section>
+      
+      <section>
         <h3>Healthcare</h3>
-        <p>Privacy is critically important in the healthcare industry. This domain looks at
-        everything from physicial interaction to connecting patients and providers with service
-        organizations.</p>
+        <p>
+          Privacy is critically important in the healthcare industry. This domain looks at
```

Comment:
This paragraph still has excess whitespace

Reply:
Repaired.